### PR TITLE
Remove weird char NBSP

### PR DESCRIPTION
**Pull request by t0t07**
The weird char is rendered as NBSP on IntelliJ

### CODE REVIEW

**Review by raboof — APPROVED** [review]
I'd be OK with also changing the fancy quotes

**Comment by YikSanChan** [comment]
@raboof thx, updated

File: akka-cluster/src/main/scala/akka/cluster/Member.scala [hunk 1]
```diff
@@ -237,37 +237,37 @@ object MemberStatus {
   @SerialVersionUID(1L) case object Removed extends MemberStatus
 
   /**
-   * Java API: retrieve the “joining” status singleton
+   * Java API: retrieve the "joining" status singleton
```
Comment (patriknw) [inline akka-cluster/src/main/scala/akka/cluster/Member.scala:240]: let's change them to
```
`Joining`, `Up`, ...
```

Comment (YikSanChan) [reply akka-cluster/src/main/scala/akka/cluster/Member.scala]: @patriknw Changed

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Pull request merged**